src/php/.php_cs.dist

```diff
@@ -0,0 +1,19 @@
+<?php
+
+$finder = PhpCsFixer\Finder::create()
+    ->exclude('bin')
+    ->exclude('ext')
+    ->exclude('tests/data')
+    ->exclude('tests/qps/generated_code')
+    ->in(__DIR__)
+;
+
+return PhpCsFixer\Config::create()
```

Comment:
Thanks for the pull request!
What do you think about changing the settings to [this config](https://github.com/GoogleCloudPlatform/php-docs-samples/blob/master/.php_cs.dist)?

Reply:
Using config file in order to ensure as far as possible not change existing code.
1. use `exclude` for not run php-fixer for generated php file by proto
2. use `setUsingCache` for re-run every time
3. use `@Symfony` and `phpdoc_no_package` and `yoda_style` for not change existing code.